src/modules/fancyzones/tests/UnitTests/ZoneWindow.Spec.cpp

```diff
@@ -1,11 +1,36 @@
 #include "pch.h"
 
+#include <filesystem>
+
 #include <lib/util.h>
 #include <lib/ZoneSet.h>
 #include <lib/ZoneWindow.h>
 #include "Util.h"
+
+#include <common/common.h>
+
 using namespace Microsoft::VisualStudio::CppUnitTestFramework;
 
+namespace
+{
+    std::wstring GenerateUniqueId(HMONITOR monitor, PCWSTR deviceId, PCWSTR virtualDesktopId) noexcept
```

Comment:
why duplicate the logic from FancyZones.cpp? Let's just make that function visible or export/link it explicitly from fancyzones.

Reply:
You're right. And since it generates unique identifier to be assigned with ZoneWindow, makes sense to move this function to `ZoneWindowUtils`.

Reply:
Done.